.github/workflows/release-code-freeze.yml

```
@@ -143,6 +144,63 @@ jobs:
+                repo: context.repo.repo,
+                title: trunkMilestone,
+              } );
+            } catch ( e ) {
+              if ( e.response?.data?.errors?.some( ( err ) => 'already_exists' === err.code ) ) {
+                console.log( `Milestone ${ trunkMilestone } already exists.` );
+              } else {
+                throw e;
+              }
+            }
+
+  cleanup-milestones:
```

actionlint (1.7.9)

job "cleanup-milestones" needs job "prepare-for-code-freeze" which does not exist in this workflow

(job-needs)